src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/InntektsmeldingRouting.kt

```diff
@@ -13,6 +13,7 @@ import no.nav.helsearbeidsgiver.auth.tokenValidationContext
 import no.nav.helsearbeidsgiver.utils.log.sikkerLogger
 
 fun Route.filtrerInntektsmeldinger(inntektsmeldingService: InntektsmeldingService) {
+    // Filterer inntektsmeldinger for et orgnr basert på request.
```

Comment:
Filterer -> Filtrerer
Tror også vi skal skrive noe ala: 
Hent inntektsmeldinger tilhørende systembrukers orgnr, filtrer basert på request
Og tilsvarende for de andre kallene, ellers kan det tolkes som at man kan spesifisere et orgnr